coverage_config_x86_64.json

```diff
@@ -1,5 +1,5 @@
 {
-  "coverage_score": 73.9,
+  "coverage_score": 86.5,
```

Comment:
Ideally this patch would have been split in 3 different patches, a patch for each of the clearly defined section. I agree that each patch would have been small but that is fine. It would have sped up the the review process by making it clear each patch is unrelated to the other. Please consider for next time.

Reply:
You mean separate patches for each of the three .rs files? Yeah, that could have been done well to make the patches smaller. Will try to do that in future.

Reply:
Right, the enhancements in each file are independent from each other and could have been split in three patches, with each patch updating its own coverage score.  I found myself looking for a relation between the hunks but there wasn't any.  But again not important enough to delay this patchset or spent any more time talking about it.